### PR TITLE
Add license to README.md

### PR DESCRIPTION
**Pull request by ashenv93**
Updated with the license.

### CODE REVIEW

**Comment by mbrandau** [comment]
You don't have to add the whole license. A little notice what license is used and a link to the file is enough.

**Comment by ashenv93** [comment]
Done. :)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # simple-chat
 A simple chat app using Angular and socket.io
+
+# License
```
Comment (mbrandau) [inline README.md]: Please make it a h2 tag by adding another #

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
...
+[MIT LICENSE](LICENSE)
+
+Copyright (c) 2017 Maximilian Brandau
```
Comment (mbrandau) [inline README.md]: You can remove this copyright notice.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,9 @@
 # simple-chat
 A simple chat app using Angular and socket.io
+
+# License
+[MIT LICENSE](LICENSE)
```
Comment (mbrandau) [inline README.md]: Add something like `This project is licensed under the terms of the [MIT License](LICENSE)`

File: README.md (hunk 1 above)
Comment (ashenv93) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (ashenv93) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (ashenv93) [reply README.md]: Done.

**Comment by mbrandau** [comment]
Closes #20

**Pull request merged**